examples/.examples-ci.yml

```diff
@@ -1,8 +1,8 @@
-.python-sdk-examples-base:
+.sdk-examples-base:
   stage: examples
   parallel:
     matrix:
-      - PYTHON_VERSION: ['3.7', '3.8', '3.9', '3.10', '3.11']
+      - PYTHON_VERSION: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```

Comment:
nit: would prefer image below to be `python:${PYTHON_VERSION}-bookworm` just to be explicit, but since the regular tag is bookworm, I'm fine leaving as is

Reply:
While we're here, sure, done.